qml/QmlAV/QmlAVPlayer.h

```diff
@@ -74,6 +74,7 @@ class QmlAVPlayer : public QObject, public QQmlParserStatus
     Q_PROPERTY(QStringList videoCodecs READ videoCodecs)
     Q_PROPERTY(QStringList videoCodecPriority READ videoCodecPriority WRITE setVideoCodecPriority NOTIFY videoCodecPriorityChanged)
     Q_PROPERTY(QVariantMap videoCodecOptions READ videoCodecOptions WRITE setVideoCodecOptions NOTIFY videoCodecOptionsChanged)
+    Q_PROPERTY(QVariantMap videoFormatOptions READ videoFormatOptions WRITE setVideoFormatOptions NOTIFY videoFormatOptionsChanged)
```

Comment:
As I can see from source codes, player.setOptionsForFormat just establishes
options for demuxer. So my questions is:
such format options as "fps" and "video_size" are parsed inside QtAV or
it's ffmpeg internal params?


Regards,
Denis D.

2016-07-27 5:01 GMT+03:00 WangBin <notifications@github.com>:

> In qml/QmlAV/QmlAVPlayer.h
> <https://github.com/wang-bin/QtAV/pull/689#discussion_r72370064>:
>
> > @@ -74,6 +74,7 @@ class QmlAVPlayer : public QObject, public QQmlParserStatus
> >      Q_PROPERTY(QStringList videoCodecs READ videoCodecs)
> >      Q_PROPERTY(QStringList videoCodecPriority READ videoCodecPriority WRITE setVideoCodecPriority NOTIFY videoCodecPriorityChanged)
> >      Q_PROPERTY(QVariantMap videoCodecOptions READ videoCodecOptions WRITE setVideoCodecOptions NOTIFY videoCodecOptionsChanged)
> > +    Q_PROPERTY(QVariantMap videoFormatOptions READ videoFormatOptions WRITE setVideoFormatOptions NOTIFY videoFormatOptionsChanged)
>
> Use formatOptions is better because you can set audio options too
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wang-bin/QtAV/pull/689/files/f66dfa7da7f8334ada54e7f03014fef35804c5ee#r72370064>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATtFPsQ35ppgMmE0uZD9x0n2NoAW9ys4ks5qZrvtgaJpZM4JVZO4>
> .
>